packages/cli/src/commands/site/version/delete/versionDeleteCommand.mts

```diff
@@ -15,17 +15,19 @@
  */
 
 import { consola } from "consola";
-import { artifacts } from "../../../net/index.mjs";
-import type { DeleteArgs } from "./DeleteArgs.js";
+import { artifacts } from "../../../../net/index.mjs";
+import type { SiteVersionArgs } from "../siteVersionArgs.js";
 
-export default async function siteDeleteCommand(args: DeleteArgs) {
+export default async function versionDeleteCommand(
+  { version, application, foundryUrl }: SiteVersionArgs,
+) {
   await artifacts.SiteAssetArtifactsService.deleteSiteVersion(
```

Comment:
It looks like some of these handwritten clients use default consola and always print out the success status code (204) on debug level. We can probably get rid of this and just rely on the shared consola with this showing up when verbose is enabled.

Reply:
Additionally, the delete here is idempotent which means if you put a non-existent version in it will always say successful not sure if this will be slightly misleading